docs/dev/table/connectors/formats/json.md

```diff
@@ -212,6 +212,53 @@ The following table lists the type mapping from Flink type to JSON type.
     </tbody>
 </table>
 
+How is json string converted to Flink SQL Row
```

Comment:
Could you add this under a "Features" section like other pages? 

Besides, we can use "Allow top-level JSON Arrays" as the sub-section title.